### ISSUE TITLE
implicit /vendor name dependency

### ISSUE THREAD

**Issue by goatherd** [reneoelke/Php54to55#18]
Not sure this is important but bootstrapping (and possibly further files) require composer vendors to be located within `/vendor`.

This however can not be enforced for composer installs.

**Comment by reneoelke** [comment]
Right, this could be a problem. But we will fix it if it's existent.
